examples/trusted_database/module_0/rust/src/database.rs

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use log::{debug, error};
+use oak::OakError;
+use oak_abi::proto::oak::application::ConfigMap;
+use serde::Deserialize;
+use trusted_database_handler::proto::oak::examples::trusted_database::{Location, PointOfInterest};
+
+/// Database structure represents internal XML fields from the following database:
+/// https://tfl.gov.uk/tfl/syndication/feeds/cycle-hire/livecyclehireupdates.xml
```

Comment:
Did you read the licensing terms? https://data.london.gov.uk/dataset/cycle-hire-availability

I think it means we need to link back to their license, right?

Reply:
Thanks! I overlooked this license first. http://www.nationalarchives.gov.uk/doc/open-government-licence/version/2/
Since it says:
```
You must, where you do any of the above:
- acknowledge the source of the Information by including any attribution statement specified by the Information Provider(s) and, where possible, provide a link to this licence;
```
Does it mean that we shouldn't use this database, or we just need to add a LICENSE file?